lib/src/client.rs

```diff
@@ -236,7 +243,9 @@ impl ClientInner {
                 let zkp_component = ZKPComponent::new(
                     blockchain_proxy.clone(),
                     Arc::clone(&network),
+                    #[cfg(feature = "zkp-prover")]
```

Comment:
Maybe we should make the feature name more consistent. In the ZKP component it's called 'prover' only. I guess going with `zkp-prover` all along is more descriptive.